test/dynamics/common.py

```diff
@@ -129,3 +201,52 @@ def setUpClass(cls):
                 import qutip  # pylint: disable=import-outside-toplevel,unused-import
         except Exception as err:
             raise unittest.SkipTest("Skipping qutip tests.") from err
+
+
+# to be removed for 0.5.0
+class TestJaxBase(unittest.TestCase):
```

Comment:
This name baffles me with ``JaxTestBase``, but if it's temporary, I guess We'll just have to put up with it.

Reply:
Yes sorry - I didn't want to change `TestJaxBase` as all of the existing testing infrastructure, which we need to work through and change, depends on it. As such I wanted to create a new base class for JAX tests. In the end yes, `TestJaxBase` will be removed.